.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "bsmetadata/vendor/dateutil"]
 	path = bsmetadata/vendor/dateutil
 	url = https://github.com/cccntu/dateutil
+[submodule "bsmetadata/vendor/REL"]
+	path = bsmetadata/vendor/REL
+	url = https://github.com/manandey/REL
```

Comment:
Out of curiosity, why did you remove this dependency from `requirements.txt` and put it here? :blush: 

After discussing it offline with @cccntu , it seems that it is complicated to pip install `bsmetadata` with this `git submodule` dependency in a folder. How do you run your code currently? 

Personally, I would be in favor of adding an `extra_require` to the `bsmetadata` setup, which has the added advantage of making the dependency optional: we only install it if we want to do the preprocessing!

This will look like this in `setup.py`:
```python
setup(
    name="bsmetadata",
    ...
    extras_require={
        "entity_preprocessing": ["REL @ git+https://github.com/manandey/REL.git#egg=REL"],
    },
)
```
By changing well on the pointer to the branch, commit of your choice. See this [link](https://stackoverflow.com/questions/32688688/how-to-write-setup-py-to-include-a-git-repository-as-a-dependency/54794506#54794506) for the syntax to use